### DISCUSSION TITLE
I occasionally have a problem starting minecraft with sodium

### DISCUSSION THREAD

**Discussion by TheMartinGamer** [FabricMC/fabric-api#3247] [Player support]
### Description

![error sodium](https://github.com/FabricMC/fabric/assets/95380103/bb471c53-2a69-48fe-a264-9be4f47f48f4)
the only mod i'm using right now for testing is sodium-fabric-mc1.19.4-0.4.10+build.24.jar
I have to clarify that when using other mods everything is perfect
This happens from one day to the next, not when I'm in the game but when I go out
I need to get this problem solved, sodium is the best way to make minecraft work for me, any help will be appreciated

**Comment by Poopooracoocoo** [comment]
What GPU and graphics driver version do you have? You want them to be up to date if you're using Sodium
